gainsboro-atelier-main/client/src/components/QuestionsAndAnswers/Answers.jsx

```diff
@@ -99,66 +96,63 @@ class Answer extends React.Component {
     }
 
     report() {
-        fetch(`/qa/answers/${this.props.id}/report`, {
-            method: 'PUT',
-            headers: {
-                'Content-Type': 'application/json',
-            },
-            body: JSON.stringify({
-                ...this.state,
-                product_id: this.props.productId
-            })
+        const Question_Id = this.props?.ans?.Question_Id;
+        const Answer_Id = this.props?.ans?.Answer_Id
+        $.post(`/answers/answer_id/report`, {
+            question_Id: Question_Id,
+            answer_Id: Answer_Id
         })
-        .then(() => {
+        .done((data) => {
+            console.log(data)
             this.setState({
                 reported: true,
             })
         })
     }
-    
+
     render() {
-        var formatedDate = new Date(this.props.date).toLocaleDateString('en-US', {
-            year: 'numeric',
-            month: 'long',
-            day: 'numeric'
-        });
+        var unix_timestamp = this.props?.ans?.Answer_Date;
+        console.log(this.props.ans)
```

Comment:
If these console.logs were used for debugging, I suggested removing them to clean up your console.